articles/cms/wordpress/extending.md

```diff
@@ -0,0 +1,228 @@
+---
+description: How to extend the Login by Auth0 WordPress Plugin with hooks, filters, and functions.
+---
+
+# Extending Login by Auth0 WordPress Plugin
+
+We're happy to review and approve new filters and actions that help you integrate even further in this plugin. Please
+ see the Contributing section on the [GitHub repo readme for this plugin](https://github.com/auth0/wp-auth0/blob/master/README.md).
+
```

Comment:
I had to stop and think for a bit here when I first read this intro. It feels like there is a short paragraph missing at the start explaining what this doc is about (start with sth like "You can create new filters and actions for your Auth0 WordPress Plugin......."). Also this doc lists a set of actions and filters so you should also talk about this in the intro (I had to scroll to see what this doc is about). So if this doc is listing the existing filters and actions say so at the beginning.